src/build/stable/commands/install_global_dependencies/install_node.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Ensure that a version argument is passed
+if [ -z "$1" ]; then
+    echo "Error: No Node.js version specified."
+    echo "Usage: ./install_node.sh <NODE_VERSION>"
+    exit 1
+fi
+
+NODE_VERSION=$1
+
+# Load nvm if it's installed
+if [ -s "$HOME/.nvm/nvm.sh" ]; then
+    source "$HOME/.nvm/nvm.sh"
+elif [ -s "/usr/local/opt/nvm/nvm.sh" ]; then
+    # Fallback for macOS/Homebrew installation path
+    source "/usr/local/opt/nvm/nvm.sh"
+else
+    echo "nvm is not installed."
+fi
+
+# Check if nvm is installed
+if ! command -v nvm &> /dev/null; then
+    echo "nvm is not installed. Installing nvm..."
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash
+
+    # Load nvm into the shell (necessary for the script to use nvm after installation)
+    export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
```

Comment:
this bit is in the nvm readme, it's not in our installation instructions for nvm... I am halfway tempted to say "hey, having nvm installed is a prerequisite" dfx is the same way. This would be just for setting node to the right version, not setting it up entirely from scratch

Reply:
Since this script is for internal use we will assume that nvm is already installed